adobe/react-spectrum
# Update Next.js config and docs for 13.1

Closes #3895

Updates example app and updates SSR docs to work with Next.js 13.1. Old way with `next-transpile-modules` is kept in the docs, but collapsed.

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Check updated config in example app.
2. Check updated docs: https://reactspectrum.blob.core.windows.net/reactspectrum/1493d243ca43ec7aa33fa8d041486c38b15604fd/docs/react-spectrum/ssr.html#nextjs

## 🧢 Your Project:
